packages/components/src/InputText/InputText.types.ts

```diff
@@ -19,29 +24,11 @@ export type InputTextVersion = 1 | 2 | undefined;
  * Do not use unless you have talked with Atlantis first.
  */
 export interface InputTextRebuiltProps
-  extends Omit<
-    React.InputHTMLAttributes<HTMLInputElement | HTMLTextAreaElement>,
-    | "onChange"
-    | "onBlur"
-    | "maxLength"
-    | "rows"
-    | "size"
-    | "suffix"
-    | "prefix"
-    | "value"
-    | "max"
-    | "min"
-    | "defaultValue"
-  > {
-  readonly error?: string;
-
-  readonly invalid?: boolean;
-  readonly identifier?: string;
-  readonly autocomplete?: boolean | AutocompleteTypes;
-  readonly loading?: boolean;
-  readonly onKeyUp?: (event: React.KeyboardEvent<HTMLInputElement>) => void;
-  readonly children?: React.ReactNode;
```

Comment:
this should never have been included. on master, trying to use this blows up because of the spreading

<img width="1327" height="536" alt="Image" src="https://github.com/user-attachments/assets/48beeac6-07af-456c-9e3f-f770976f5a24" />